plugin.xml

```diff
@@ -22,7 +22,7 @@
 xmlns:android="http://schemas.android.com/apk/res/android"
 xmlns:rim="http://www.blackberry.com/ns/widgets"
            id="cordova-plugin-media-capture"
-      version="2.0.1-dev">
+      version="2.0.0">
```

Comment:
This change and probably most of the others in this file should be removed before merging. Just the last one seems to be a real change from this PR.